packages/@aws-cdk-testing/framework-integ/test/aws-codepipeline-actions/test/integ.cfn-template-from-repo.lit.ts

```diff
@@ -52,6 +52,7 @@ const prodStage = {
 };
 
 new codepipeline.Pipeline(stack, 'Pipeline', {
+  crossAccountKeys: true,
```

Comment:
Why do we need to explicitly set it to `true`? The feature flag is not enabled by default and shouldn't change existing behaviour right? 
Same questions for all other places where you added this prop.

Reply:
It appears that these feature flags are enabled for integ tests in CodeBuild CI. I put it all back and got an error in CodeBuild CI.

[Changes](https://github.com/aws/aws-cdk/pull/28556/commits/08642e67bb15c3063911ffa29516781e606c6cfa)

[CodeBuild CI](https://ops-cdk-autobuild-prs-autobuildv2github-buildlogs-g9jsj1nfmnjp.s3.amazonaws.com/23ec2db7-fc66-419c-ab86-c5174e5011b3/build.log?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAWYORDRBKHNMYQ2GP%2F20240201%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20240201T080624Z&X-Amz-Expires=600&X-Amz-SignedHeaders=host&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEHkaCXVzLWVhc3QtMSJHMEUCICQtWda4Tw8FRPOinUi81P%2FPy12z5DXRV0%2BVbOMJPWoEAiEAmH7xJLOUMx0w%2FapIn2MbmreGiL%2FyuAzELEAi7Rplio0q0wMIQRACGgw0NjQ4MzE4NzUxNTYiDGSQ9U8LQfZ%2FRz65NSqwA0Cjj2itZGvU7iXRONoepNq2ILT5AsJnkCKDU8Rneb2rmmN3NWb0RQQ9ckWMj1XcssjyZUGh0rgULxyxFB5%2Fx%2FZzoT1IJsk63VgFtKu8E6mQwrPIntubTJhVBAyt%2B5wApHg5HG3OVz%2Fwd1VV0Jt9WvElWvBdTVmAZO%2FrVhaaSEG4coB4zlF5xHMWPJpKxFB7mkuuRCTKDDDtSPKVGbASlJ7D12zZt23DQyMT85WQu9kUn7Z4NigRjr8HSgQHYf0pB8%2B0g4voKD1iut70CrJxW7dHVniZvkJZc9EXL2pIOyRtLnHORUjGqlr5V4ZCXsiACQwHhSdcT6CD%2BfLFhYv6YMeksy6DP7KhAGwy6v5bH4ty4V23Ym14a4xoX%2BiQWx92aDrO3FDmzvW5skkye%2FjOX25ZcDoMcr93%2FATi4ony8cUy3kE0z3r0cUEXxcNwxMDS4Uj7LRhwOpSIc9ttZ8HBAqnslKZGRVNJTsMbpmag%2BpMQ0Z973dJ9nMRsdELbyuLSPGrUbtdar%2F8LTUa5bNaUpv3cO0%2F9bN3JCt6iJc3gDgBPk8ciLzzyl5tMtV7844M1ezD%2Boe2tBjqeARiPAg0JkCQgZ%2Fe9vWlGVutYkQm%2Fx%2FTHr%2FMOts%2F4C9aBiyM6K5iKx3hkidPyREnI30V0XHoaYq1mnlA9Bv8FkLht%2BAe4aMmnWtF1dyMfA5ku%2F5UBujDeByH7FFKWMoQ%2BRJFJhvbBslhqLDCLoQ%2BMWZiT1cUJbEOJqm15agf1z%2BMGsS%2F5i21bQDF%2BAF%2BiL6eWLQvAtey3KxwiD%2Fyp2FOs&X-Amz-Signature=ba3e29ab3f83d46da1e8b3b6b792efe32ce6f61dd813b5e4646b2a3c5d9f78fb)

Reply:
I still tried to go back to the original code to test it out, and the CI was an error.

https://github.com/aws/aws-cdk/pull/28556/commits/fd0ee48d2cadccaeabb31e51e0dcc75cb1ac41d2

![ci1](https://github.com/aws/aws-cdk/assets/24818752/636a24e7-5061-4256-9019-fb990f165ac7)

![ci2](https://github.com/aws/aws-cdk/assets/24818752/9db9a4cb-361b-4ddf-bf04-22634250b300)

However, there is also an integ test where the function flag remains off by default.

https://github.com/go-to-k/aws-cdk/blob/cba7c75fdd055b1cd48f21fbaa3ab29cf5944693/packages/@aws-cdk-testing/framework-integ/test/aws-cloudwatch-actions/test/integ.lambda-alarm-action.ts#L60-L74

I thought it might be due to a difference in integ.json ([this file](https://github.com/go-to-k/aws-cdk/blob/cba7c75fdd055b1cd48f21fbaa3ab29cf5944693/packages/@aws-cdk-testing/framework-integ/test/aws-codepipeline-actions/test/integ.cfn-template-from-repo.lit.js.snapshot/integ.json) by `new integ.IntegTest()` and [another file](https://github.com/go-to-k/aws-cdk/blob/cba7c75fdd055b1cd48f21fbaa3ab29cf5944693/packages/@aws-cdk-testing/framework-integ/test/aws-cloudwatch-actions/test/integ.lambda-alarm-action.js.snapshot/integ.json)), but since I still don't understand how this works, I think I'll leave this PR as is for now. (Because fixing this could be another potentially destructive change.)

Reply:
I ran the other test (integ.lambda-alarm-action.ts) as a new integ file (with no existing snapshot) to try it out, and it seems that the feature flag is enabled by default.
(Does using `new integ.IntegTest()` turn off the feature flag with an existing snapshot?)

Either way, I think we're good to go for now.